tfjs-core/src/ops/to_pixels_test.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2020 Google Inc. All Rights Reserved.
+ * Copyright 2023 Google Inc.
```

Comment:
Do we need to modify it if we only modify the content, it is not a new file?

Reply:
I  donot know. I revert this change.